changelogs/unreleased/3952-syntax-highlight-elif-vim.yml

```
@@ -0,0 +1,3 @@
+description: Add highlighting for the elif keyword for vim
+change-type: minor
```

```suggestion
change-type: patch
```
Missed this earlier. Approving already because it's not too bad if you miss this comment.